Pulse.Core/Calls/ConnectionInfo.cs

```diff
@@ -1,3 +1,9 @@
 namespace Pulse.Core.Calls;
 
-internal record ConnectionInfo(string RemoteIPAddress, int MinPort, int MaxPort);
\ No newline at end of file
+internal record ConnectionInfo(
+    string IPAddress,
+    int MinPort,
+    int MaxPort,
+    byte[] PublicKey,
+    byte[]? IV
```

Comment:
```suggestion
    byte[]? IV = null
```

Reply:
We should separate it into two different records.